tests/phpstan/bootstrap.php

```diff
@@ -1,6 +1,6 @@
 <?php
 
-define( 'WP_PLUGIN_CHECK_PLUGIN_DIR_PATH', '' );
-define( 'WP_CONTENT_DIR', '' );
-define( 'WP_PLUGIN_DIR', '' );
-define( 'WP_PLUGIN_CHECK_PLUGIN_DIR_URL', '' );
+defined( 'WP_PLUGIN_CHECK_PLUGIN_DIR_PATH' ) or define( 'WP_PLUGIN_CHECK_PLUGIN_DIR_PATH', '' );
+defined( 'WP_CONTENT_DIR' ) or define( 'WP_CONTENT_DIR', '' );
+defined( 'WP_PLUGIN_DIR' ) or define( 'WP_PLUGIN_DIR', '' );
+defined( 'WP_PLUGIN_CHECK_PLUGIN_DIR_URL' ) or define( 'WP_PLUGIN_CHECK_PLUGIN_DIR_URL', '' );
```

Comment:
That's really not the way to do it. This file is for making PHPStan aware of WP-specific constants.

Since https://github.com/szepeviktor/phpstan-wordpress/pull/193 it already knows about `WP_PLUGIN_DIR` so we can simply remove the definition here.


```suggestion
define( 'WP_PLUGIN_CHECK_PLUGIN_DIR_PATH', '' );
define( 'WP_CONTENT_DIR', '' );
define( 'WP_PLUGIN_CHECK_PLUGIN_DIR_URL', '' );
```